testdata/p4_16_samples/struct_with_st-bmv2.p4

```diff
@@ -0,0 +1,53 @@
+#include "v1model.p4"
+struct metadata {
+    /* empty */
+}
+
+enum bit<7> X {
+    Zero = 0,
+    One = 1
+}
+
+header BITS {
+    bit<8> bt;
+    int<8> it;
+    bool b;
+    X  x; 
+}
+
+struct headers {
+    BITS    bits;
+    bit<8>  bt;
+    int<8>  it;
+    bool    b;
+    X       x;
+}
+
+parser MyParser(packet_in packet,
+                out headers hdr,
+                inout metadata meta,
+                inout standard_metadata_t standard_metadata) {
+    state start {
+        packet.extract(hdr.bits);
+        hdr.bt = hdr.bits.bt;
+        hdr.it = hdr.bits.it;
+        hdr.b = hdr.bits.b;
+        hdr.x = hdr.bits.x;
+        transition accept;
+    }
+}
+
+control mau(inout headers hdr, inout metadata meta, inout standard_metadata_t sm) {
```

Comment:
can you please use the fields in the hdr here to make sure that they are correctly passed from the parser to the ingress?
Do something with them which influences the visible output.